Player.gd

```diff
@@ -21,6 +21,8 @@ onready var animation_player = $AnimationPlayer
 onready var animation_tree = $AnimationTree
 onready var animation_state = animation_tree.get("parameters/playback")
 
+signal collided
```

Comment:
This is new to me. What is a signal do?

Reply:
There is a tab on the right called "Node" and its where you can set up events/signals to be accessed by other objects.
declaring the signal adds it to the node list then you can go to the object listning for the signal and add the method for it to call by clicking on the node item.

Reply:
Cool.